e2e/pages/events/enterOthersEvent.page.js

```diff
@@ -23,22 +18,20 @@ module.exports = {
       litigationIssuesDetails: `#others_${index}_litigationIssuesDetails`,
       detailsHidden: (option) => {
         return {
-          option: `#others_${index}_detailsHidden-${option}`,
+          option: `#others_${index}_detailsHidden_${option}`,
           reason: `#others_${index}_detailsHiddenReason`,
         };
       },
     };
   },
 
   async enterOtherDetails(other) {
-    const elementIndex = await this.getActiveElementIndex();
+    const elementIndex = await this.getActiveElementSelector();
 
     await I.runAccessibilityTest();
     I.fillField(this.fields(elementIndex).name, other.name);
-    I.click(this.fields(elementIndex).DOB.day);
-    I.fillField(this.fields(elementIndex).DOB.day, other.DOB.day);
-    I.fillField(this.fields(elementIndex).DOB.month, other.DOB.month);
-    I.fillField(this.fields(elementIndex).DOB.year, other.DOB.year);
+    const dateSelector = `(//*[@id="DOB"])[${await this.getActiveElementIndex()}]`;
```

Comment:
again, is it worth overwriting the `this.fields(elementIndex).DOB` value with this new xpath? to follow the existing pattern?